app/src/main/java/com/concordia/mcga/factories/IndoorMapFactory.java

```diff
@@ -0,0 +1,37 @@
+package com.concordia.mcga.factories;
+
+import static java.lang.Integer.parseInt;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import com.concordia.mcga.activities.R;
+import com.concordia.mcga.models.IndoorMap;
+import com.concordia.mcga.utilities.pathfinding.TiledMap;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class IndoorMapFactory {
+    public static IndoorMap getHall4thFloor(Context context) {
+        return getIndoorMap(context, R.raw.hall4);
+    }
+
+    @NonNull
+    private static IndoorMap getIndoorMap(Context context, int rawResourceId) {
+        TiledMap map = new TiledMap(350,317);
```

Comment:
arbitrary. was waiting for actual values from @Vynny. Now that I got it, can update